kuma/wiki/tasks.py

```diff
@@ -460,3 +461,31 @@ def tidy_revision_content(pk):
             )
         # return the errors so we can look them up in the Celery task result
         return errors
+
+
+@task(rate_limit='10/m')
+def set_share_url(pk):
```

Comment:
please use a more descriptive name for the task such as `update_document_share_url` or something. the name of the ask will show up in the celery admin and it's then easier to distinguish from other the tasks